### PR TITLE
feat: Sync skip delete

### PR DESCRIPTION
**Pull request by murali-shris**
**- What I did**
- introduced a flag skipDeletes in sync. 
**- How I did it**
- modified syntax of sync:from verb
- modified impl of sync verb handler to include skipDeletes flag.
**- How to verify it**
- tests should pass

### CODE REVIEW

**Review by gkc — APPROVED** [review]

File: packages/at_commons/pubspec.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 name: at_commons
 description: A library of Dart and Flutter utility classes that are used across other components of the atPlatform.
-version: 5.0.1
+version: 5.0.2
```
Comment (gkc) [inline packages/at_commons/pubspec.yaml]: should be 5.1.0 as this is new feature

**Review by gkc — CHANGES_REQUESTED** [review]

File: packages/at_commons/pubspec.yaml (hunk 1 above)
Comment (murali-shris) [reply packages/at_commons/pubspec.yaml]: Done

File: packages/at_commons/CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,5 @@
+## 5.1.0
+- feat: Introduce skipDeletes for sync:from verb
```
Comment (gkc) [inline packages/at_commons/CHANGELOG.md]: skipDeletesUntil

File: packages/at_commons/lib/src/verb/sync_verb_builder.dart [hunk 3]
```diff
@@ -9,6 +9,8 @@ class SyncVerbBuilder implements VerbBuilder {
 
   bool isPaginated = false;
 
+  int? skipDeletesUntil;
```
Comment (gkc) [inline packages/at_commons/lib/src/verb/sync_verb_builder.dart:19]: Please take this opportunity to add `///` documentation for all of the parameters (commitId, regex, limit, isPaginated, skipDeletesUntil)

**Review by gkc — CHANGES_REQUESTED** [review]

File: packages/at_commons/CHANGELOG.md (hunk 2 above)
Comment (gkc) [reply packages/at_commons/CHANGELOG.md]: @murali-shris `skipDeleteUntil`

Comment (murali-shris) [reply packages/at_commons/CHANGELOG.md]: Done

File: packages/at_commons/lib/src/verb/sync_verb_builder.dart (hunk 3 above)
Comment (murali-shris) [reply packages/at_commons/lib/src/verb/sync_verb_builder.dart:19]: added the dart docs

**Review by gkc — APPROVED** [review]

**Pull request merged**